client/src/components/DetailedCard/DetailedCardForm.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import { Box, Button, Grid, Typography } from '@material-ui/core';
+import { Link, BrowserRouter as Router } from 'react-router-dom';
+import useStyles from './useStyles';
+import MenuBook from '@material-ui/icons/MenuBook';
+import CloseIcon from '@material-ui/icons/Close';
+import AccessTime from '@material-ui/icons/AccessTime';
+import ChatBubbleOutline from '@material-ui/icons/ChatBubbleOutline';
+const DetailedCardForm: React.FC = () => {
+  const classes = useStyles();
+  const [addCard, setAddCard] = useState(['Tag', 'Check-list', 'Deadline', 'Attachement', 'Cover']);
+  const [actionCard, setActionCard] = useState(['Move', 'Copy', 'Share', 'Delete']);
+  return (
+    <Box>
+      <Grid container xs={12} className={classes.container} direction="row">
+        <Grid container item xs={9} direction="column">
+          <Box className={classes.itemBox}>
+            <Typography className={classes.boxContent}>
+              <div>
+                <MenuBook className={classes.cardIcons} />
+              </div>
+              Description:
+            </Typography>
+            <Box className={classes.boxItems}>
+              <textarea
+                name="description"
+                rows={3}
+                cols={57}
+                className={classes.textArea}
+                placeholder="Write a description"
+              />
```

Comment:
You can use MUI `TextField` with `multiline` prop for `<textarea>`

Reply:
Changed.
Thanks for letting me know about it.